src/NHibernate/Engine/ActionQueue.cs

```diff
@@ -107,17 +109,17 @@ public void AddAction(BulkOperationCleanupAction cleanupAction)
 		{
 			RegisterCleanupActions(cleanupAction);
 		}
-		
```

Comment:
While working, I have ended up CTRL+K CTRL+D due to wrecked indentation in Java ported code for the fix, without noticing the amount of bad spaces already littering this file. Actual changes start at ligne 507.